PowerShell/JumpCloud Module/Tests/Private/Policies/Convert-RegToPSObject.Tests.ps1

```diff
@@ -0,0 +1,40 @@
+Describe -Tag:('JCPolicy') 'Registry File Tests' {
+    Context 'Test Reg File Conversion' {
+        $regFile = Convert-RegToPSObject -regFilePath $PesterParams_RegistryFilePath
+        It 'Convert-RegToPSObject should return object with values' {
+            foreach ($regKey in $regFile) {
+                $regKey.customLocation | Should -Not -BeNullOrEmpty
+                $regKey.customValueName | Should -Not -BeNullOrEmpty
+                $regKey.customRegType | Should -Not -BeNullOrEmpty
+                $regKey.customData | Should -Not -BeNullOrEmpty
+            }
+        }
+        It 'Convert-RegToPSObject validate correct values' {
+            foreach ($regKey in $regFile) {
+                $regKey.customLocation | Should -Be "SOFTWARE\Policies\Microsoft\Power\PowerSettings"
+                switch ($regKey.customRegType) {
+                    DWORD {
+                        $regKey.customValueName | Should -Be "DWORDValue"
```

Comment:
thanks for the different tests here, I think it's good to capture our different values